spring-integration-jdbc/src/main/java/org/springframework/integration/jdbc/lock/JdbcLockRegistry.java

```diff
@@ -131,7 +132,7 @@ public void lock() {
 					}
 					break;
 				}
-				catch (TransientDataAccessException e) {
+				catch (TransientDataAccessException | TransactionTimedOutException e) {
```

Comment:
You know that we can't use this structure for `4.3.x`. That one is still based on Java 6.
Saying that it raises a question to you: why do you ask for `4.3.x` if you don't use Java 6 any more?

Reply:
Ho crap sorry..

We keep 4.3.x because of spring version. We not upgrade the project to a 5+ spring version yet :/

Reply:
I have to create a new pull request for `4.3.x`? I'm not familiar with your usage. I'm so sorry

Reply:
No, you don't: I'll back port it properly. Just curious why do you need fix in the version for Java 6, when you definitely could move already to `5.x` generation if you don't use Java 6 any more...